gene_finder.py

```diff
@@ -2,7 +2,7 @@
 """
 YOUR HEADER COMMENT HERE
```

Comment:
For the upcoming mini projects, please add your name and a short description of the project.